documentation/create_task/define_task.qmd

```diff
@@ -34,21 +34,28 @@ knitr::asis_output(md)
 
 ## Step 3: Create task info
 
-Now create a task info metadata file at `src/tasks/<task_id>/api/task_info.yaml`:
+Now create a task info metadata file in the `src/tasks/<task_id>/api` directory. This file will be part of the automatic "readme" creation in the repo directory of this task. The task info file should be named `task_info.yaml` and should contain the following information:
 
 ```{.yaml filename="src/tasks/<task_id>/api/task_info.yaml"}
-task_id: your_task_id
-task_name: Your task name
-summary: "FILL IN: A one line description of what this purpose of this task is."
-description: |
-  FILL IN: The task description and motivation you created earlier.
-authors:
-  - name: "Your name"
-    roles: [ author, maintainer ]
-    props:
-      github: your-github-account
-      orcid: "0000-1111-2222-3333"
+task_id: your_task_id # <1>
+task_name: Your task name # <2>
+summary: "FILL IN: A one line description of what this purpose of this task is." # <3>
+description: | # <4>
+  FILL IN: The task description and motivation you created earlier. # <4>
+authors: # <5>
+  - name: "Your name" # <5>
+    roles: [ author, maintainer ] # <5>
+    info: # <5>
+      github: your-github-account # <5>
+      orcid: "0000-1111-2222-3333" # <5>
```

Comment:
I think this is no longer accurate, see https://github.com/openproblems-bio/openproblems-v2/blob/main/src/common/schemas/task_info.yaml.

The `defs_common.yaml#/definitions/Name` refers to https://github.com/openproblems-bio/openproblems-v2/blob/main/src/common/schemas/defs_common.yaml#L75-L80C18